astroid/bases.py

```diff
@@ -41,7 +41,6 @@
 
 # TODO: check if needs special treatment
 BOOL_SPECIAL_METHOD = "__bool__"
```

Comment:
This is perhaps out of scope, but I noticed today that we should probably hardcode this. It's only used once.